airflow-core/src/airflow/exceptions.py

```diff
@@ -46,26 +42,24 @@ def serialize(self):
         return f"{cls.__module__}.{cls.__name__}", (str(self),), {}
 
 
-class AirflowBadRequest(AirflowException):
-    """Raise when the application or server cannot handle the request."""
-
-    status_code = HTTPStatus.BAD_REQUEST
+class TaskNotFound(AirflowException):
```

Comment:
I have a feeling, that everything in core like this could now extend from Exception/RuntimeException, as we don't have many cased in core we do an `except AirflwoException`, so we could change them and then _this_ AirflowException could be a lazy import/compat shim to airflow.sdk.exceptions.AirflowException?